### PR TITLE
Improved wording

### PR DESCRIPTION
**Pull request by fgourichon**

### CODE REVIEW

File: core/config/initializers/00_tenant_config_schema.rb [hunk 1]
```diff
@@ -340,81 +352,109 @@
...
         },
+        apps_management: {
+          type: "object",
+          title: "Apps management",
+          properties: {
+            enabled: {
+              type: "boolean",
+              default: true,
+              title: "Enabled",
+              description: "Let you filter apps which are available in the marketplace and in the admin panel"
```
Comment (ouranos) [inline core/config/initializers/00_tenant_config_schema.rb]: Allow you to manage your customer's app (add apps / remove apps)

File: core/config/initializers/00_tenant_config_schema.rb [hunk 2]
```diff
@@ -268,19 +273,22 @@
...
         organization_management: {
           type: "object",
+          title: "Organization management",
           properties: {
             enabled: {
               type: "boolean",
               default: true,
-              description: "Allow user to create and manage Organizations",
+              title: "Enabled",
+              description: "Allow your customer to create and manage multiple companies",
```
Comment (ouranos) [inline core/config/initializers/00_tenant_config_schema.rb]: This also disable:
- creation of new company
- the company settings menu on the left side

File: core/config/initializers/00_tenant_config_schema.rb [hunk 3]
```diff
@@ -461,17 +505,24 @@
             enabled: {
               type: "boolean",
               default: true,
-              description: "Enable staff management"
+              title: "Enabled",
+              description: "Enable staff management, ie. the possibility to add admin users with limited rights"
```
Comment (ouranos) [inline core/config/initializers/00_tenant_config_schema.rb]: You can also add "admin" users with this feature

**Comment by fgourichon** [comment]
Comments addressed except the 'empty objects as section'. To be investigated

**Pull request merged**